docs/tutorial/gateway-protocol

```diff
@@ -0,0 +1,255 @@
+  const fragment = `eth-key${Date.now()}`;
+  const newKeyVerificationMethod = {
+    flags: [BitwiseVerificationMethodFlag.CapabilityInvocation],
+    fragment,
+    keyData: Buffer.from(utils.arrayify(newEthKey.address)),
+    methodType: VerificationMethodType.EcdsaSecp256k1RecoveryMethod2020,
+  };
+
+  // Set VM
+  await didSolService.addVerificationMethod(newKeyVerificationMethod).withAutomaticAlloc(authority.publicKey).rpc();
+
+  // Set Ownership flags
+  await didSolService
+    .setVerificationMethodFlags(fragment, [
+      BitwiseVerificationMethodFlag.CapabilityInvocation,
+      BitwiseVerificationMethodFlag.OwnershipProof,
+    ])
+    .withEthSigner(newEthKey)
+    .rpc();
+};
+```
+
+1. Import the addEthKeyWithOwnershipToDID function and the necessary dependencies (e.g. Wallet, DidSolIdentifier, DidSolService, EthWallet, etc.) at the top of your file.
+2. Create a new Wallet object representing the authority that will be used to create the new DID.
+3. Call the addEthKeyWithOwnershipToDID function, passing in the authority object as an argument.
+4. This will create a new DID based on the authority object's publicKey, and add a new Ethereum key with ownership proof to it. The updated DID will then be sent to an RPC endpoint.
+5. You can now use the updated DID and its associated Ethereum key for any operations that require ownership proof.
+
+Fifth, we will add service to the user's DID.
+
+```ts
+
+//addServiceToDID function which takes in a Wallet object representing an authority, and a Service object, and adds the service to the authority's DID using the didSolService.addService method.
+
+export const addServiceToDID = async (authority: Wallet, service: Service) => {
+  const did = DidSolIdentifier.create(authority.publicKey, cluster);
+  const didSolService = DidSolService.build(did, {
+    wallet: authority,
+  });
+
+  await didSolService.addService(service).withAutomaticAlloc(authority.publicKey).rpc();
+};
+```
+
+1. Import the addServiceToDID function and the necessary dependencies (e.g. Wallet, DidSolIdentifier, DidSolService, Service, etc.) at the top of your file.
+
+2. Create a new Wallet object representing the authority that will be used to add the service to the DID.
+
+3. Create a new Service object representing the service that you want to add to the DID.
+
+4. Call the addServiceToDID function, passing in the authority and service objects as arguments.
+
+5. This will add the service to the DID associated with the authority object, and then send the updated DID to an RPC endpoint.
+
+6. You can now use the updated DID with the added service for any operations that require that service.
+
+Once we have done all the steps, we will have a gateway pass issued to the user.
+
+```ts
+
+(async () => {
+  const connection = new Connection(clusterApiUrl(cluster));
+  // const connection = new Connection("https://rough-misty-night.solana-devnet.quiknode.pro/b57300ff234c12e95763e9b8cda67e9d86772a0d/")
```

Comment:
I don't think we should have commented out code in the tutorial. I think it is a bit confusing. Some below, with `authority` and the airdrops